src/pages/Account/Components/AccountAllTransactions.tsx

```diff
@@ -26,6 +27,12 @@ function RenderPagination({
   ) => {
     searchParams.set("page", newPageNum.toString());
     setSearchParams(searchParams);
+
+    // logging
+    Statsig.logEvent("go_to_new_page", newPageNum, {
```

Comment:
do we wanna have a separate constants file to keep track of all log events for statssig?

Reply:
also will we have duplicated logs all over the place if we don't put this piece of logics in `useEffect`? with every re-render, this event will be logged?

Reply:
yes it would be best to have a constants file to track them. this PR is for trying statsig out so we can maybe think more about this later.

Reply:
It won't have duplicated logs because the log will only be triggered when user clicks on a page number and goes to a new page, which is the expected behaviour.